### ISSUE TITLE
Character journal replaces 9th slot upon login

### ISSUE THREAD

**Issue by GetShot** [DestinyNetwork/DungeonRealms-Bug-Report#144]
Whenever a server crashes or you logout and back in, if your character journal isn't in your 9th hotbar slot then most of the time it'll get moved to your 9th hot bar slot wiping whatever was there previously.

**Comment by ImprudentSolid0** [comment]
Yeah. I lost an Orb of Alteration because the server crashed and my book was in slot 9.
